### ISSUE TITLE
vmagent v1.62.0 wierd /targets output.

### ISSUE THREAD

**Issue by ThomasADavis** [VictoriaMetrics/VictoriaMetrics#1408]
**Describe the bug**
```
[root@mouse-r13 ~]# curl http://10.42.0.17:8429/targets
job="vmagent-omni"(1/1up)
	state=up, endpoint=http://10.42.0.17:8429/metrics,{ %space %}labels={cluster="dev-k3s",endpoint="http",instance="10.42.0.17:8429",job="vmagent-omni",namespace="monitoring",pod="vmagent-omni-bfdfdd447-wxh97",prometheus="monitoring/omni",service="vmagent-omni"}, last_scrape=3.402s ago, scrape_duration=0.003s, samples_scraped=585, error=""
job="monitoring/vmagent-omni/0"(0/0 up)
```

what is the { %space %} doing in front of the labels tag?

**To Reproduce**
install v1.62 via vm-operator, using tag v1.62.0

**Expected behavior**

no extra characters in output.

**Comment by f41gh7** [comment]
It's a typo. Thanks for reporting. Will be fixed.

**Comment by ThomasADavis** [comment]
I assume this is purely cosmetic then.

**Comment by valyala** [comment]
The `{ %space %}` typo at `/targets` page has been fixed in the commit 1b0501a09eb8931d45a19bb8a4e8a27dd4f0ba9d . This commit will be included in the next release.

**Comment by valyala** [comment]
The bugfix has been included in [v1.63.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed.
